Assets/_Functionalities/ObjectLibrary/Scripts/ObjectLibraryButton.cs

```diff
@@ -77,7 +78,7 @@ protected virtual void CreateObject()
 
         private void SpawnAtOpticalPosition()
         {
-            var opticalRaycaster = FindAnyObjectByType<OpticalRaycaster>();
+            var opticalRaycaster = ServiceLocator.GetService<OpticalRaycaster>();
```

Comment:
Kan je deze changes voor nu reverten? Dit conflicteert met mijn PR en is het flink aangepast 😅